docs/samplesheets/fromSamplesheet.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Create a channel
+description: Channel factory to create a channel from a sample sheet.
+---
+
+# Create a channel from a sample sheet
+
+## `fromSamplesheet`
+
+This function validates and converts a samplesheet to a ready-to-use Nextflow channel.
+This is done using information encoded within a sample sheet schema (see the [docs](../nextflow_schema/sample_sheet_schema_specification.md)).
+
+The function has one mandatory argument: a string specifying the name of the parameter used to provide a samplesheet.
+This parameter be described in the Nextflow parameter schema using as a file, with a `schema` field:
+
+```json hl_lines="4"
+{
+  "type": "string",
+  "format": "file-path",
+  "schema": "assets/foo_schema.json"
+}
+```
+
+The given sample sheet schema specified in the `schema` key is then loaded and used for validation and sample sheet generation.
+
+Two additional function optional arguments can be used:
+
+- File name for the pipeline parameters schema. (Default: `nextflow_schema.json`)
+- Whether to create the meta map as an `ImmutableMap` - see [docs](immutable_map.md) (Default: `true`)
+
+```groovy
+Channel.fromSamplesheet('input')
+```
+
+```groovy
+Channel.fromSamplesheet(
+  'input',
+  schema_filename: 'custom_nextflow_schema.json',
+  immutable_meta: false
+)
+```
+
+## Basic example
+
+In [this example](../../examples/fromSamplesheetBasic/), we create a simple channel from a CSV samplesheet.
+
+```
+--8<-- "examples/fromSamplesheetBasic/log.txt"
+```
+
+=== "main.nf"
+
+    ```groovy
+    --8<-- "examples/fromSamplesheetBasic/pipeline/main.nf"
+    ```
+
+=== "samplesheet.csv"
+
+    ```csv
+    --8<-- "examples/fromSamplesheetBasic/samplesheet.csv"
+    ```
+
+=== "nextflow.config"
+
+    ```groovy
+    --8<-- "examples/fromSamplesheetBasic/pipeline/nextflow.config"
+    ```
+
+=== "nextflow_schema.json"
+
+    ```json hl_lines="19"
+    --8<-- "examples/fromSamplesheetBasic/pipeline/nextflow_schema.json"
+    ```
+
+=== "assets/schema_input.json"
+
+    ```json
+    --8<-- "examples/fromSamplesheetBasic/pipeline/assets/schema_input.json"
+    ```
+
+## Order of fields
+
+[This example](../../examples/fromSamplesheetOrder/) demonstrates that the order of columns in the sample sheet file has no effect.
+It is the order of fields in the sample sheet schema which defines the order returned by `fromSamplesheet()`.
```

Comment:
can bold or emphasize this!

Reply:
Agree, this needs hammering home. I've added a `Danger` admonition.